contracts/flatten/ServiceRegistry-flatten.sol

```diff
@@ -1955,13 +1955,12 @@ contract ServiceRegistryProxy {
     /// @dev Creates multisig instance controlled by the set of service agent instances and deploys the service.
     /// @param serviceOwner Individual that creates and controls a service.
     /// @param serviceId Correspondent service Id.
-    /// @param multisigImplementation Multisig implementation address.
     /// @param data Data payload for the multisig creation.
     /// @return multisig Address of the created multisig.
     function deploy(
         address serviceOwner,
         uint256 serviceId,
-        address multisigImplementation,
+        address,
```

Comment:
Just to get rid of the warning.

Reply:
What warning?

Reply:
Of the unused variable. It's a flattened contract with several predefined addresses.